src/Jira/Api.php

```diff
@@ -61,9 +60,9 @@ class Api
     protected $statuses;
 
     /**
-     * create a jira api client.
+     * Create a JIRA API client.
      *
-     * @param $endpoint
+     * @param string $endpoint URL
      * @param AuthenticationInterface $authentication
```

Comment:
Any purpose in not adding parameter description to other method parameters?

Same is true for other DocBlocks.